### ISSUE TITLE
Add TypeScript support

### ISSUE THREAD

**Issue by belozer** [jeetiss/tabs#32]
Because typing is mainstream now

**Comment by Shaker-Hamdi** [comment]
Yup, I just installed this package and was ready to use it just to find out there are no types for it. So it would be great to add TypeScript support.

**Comment by sashayudaev** [comment]
@Shaker-Hamdi why use react-tabs if you could temporarily add your own typings? The library is rather small so it shouldn't be lots of work.

**Comment by Shaker-Hamdi** [comment]
@sashayudaev When you're in a hurry to meet the deadline, any amount of extra work will hurt you. Plus I'm not that fond of Typescript to begin with :D.
So the idea of typing something that I didn't build myself and figuring out the types is something that gives me anxiety just bu thinking about it :D
